src/core/base-layout/components/index.ts

```diff
@@ -1,3 +1,3 @@
-export { Footer } from './footer';
-export { Header } from './header';
-export { Partnered } from './partnered';
+export { Footer } from '../../../widgets/footer';
+export { Header } from '../../../widgets/header';
+export { Partnered } from '../../../widgets/partnered';
```

Comment:
can we use alias import here?
```suggestion
export { Footer } from '@/widgets/footer';
export { Header } from '@/widgets/header';
export { Partnered } from '@/widgets/partnered';
```

Reply:
@YulikK ✅done